packages/landing/scripts/deploy.js

```diff
@@ -40,36 +41,45 @@ async function deploy() {
     if (s3Bucket == null) throw new Error('Failed to locate edge bucket in current account');
 
     let hasChanges = false;
-    const files = await fs.readdir(DistDir);
-    for (const fileName of files) {
-        const fileData = await fs.readFile(`${DistDir}/${fileName}`);
-        const hash = crypto.createHash('sha512').update(fileData).digest('base64');
 
-        const oldHash = await getHash(s3BucketName, fileName);
-        // Only upload files if they have changed
-        if (oldHash == hash) {
-            console.log('Skipped', fileName, `${(fileData.byteLength / 1024).toFixed(2)}Kb`, hash);
-            continue;
-        }
-        hasChanges = true;
+    /**
+     * Upload all files from the dist directory, including any sub directories, to S3 bucket .
+     */
+    const uploadDirectory = async () => {
```

Comment:
NIT: Why make a function and call it directly after? could you not just execute the function